DXMainClient/DXGUI/Multiplayer/GameListBox.cs

```diff
@@ -32,13 +32,12 @@ public class GameListBox : XNAListBox
 
         public List<GenericHostedGame> HostedGames;
 
-        private double _gameLifetime = 35.0;
+        public double GameLifetime { get; set; } = 35.0;
 
-        public double GameLifetime
-        {
-            get { return _gameLifetime; }
-            set { _gameLifetime = value; }
-        }
+        /// <summary>
+        /// A predicate for setting a filter expression for displayed games.
+        /// </summary>
+        public Predicate<GenericHostedGame> GameMatchesFilter { get; set; }
```

Comment:
Since this is never modified once assigned, nor is there any need to be able to access it from the outside, I think it'd be better to make it private and readonly and deliver it in the constructor.

Reply:
`readonly` is not available in this case, made it private though.

Reply:
@Metadorius You can make it readonly (assignable only from the constructor) by removing the `set` accessor entirely.